compiler/rustc_feature/src/active.rs

```diff
@@ -161,6 +161,8 @@ declare_features! (
     (active, profiler_runtime, "1.18.0", None, None),
     /// Allows using `rustc_*` attributes (RFC 572).
     (active, rustc_attrs, "1.0.0", None, None),
+    /// Allows using internal rustdoc features like `doc(primitive)` or `doc(keyword)`.
+    (active, rustdoc_internals, "1.58.0", Some(90418), None),
```

Comment:
This is in the wrong section; it should be next to doc_notable_trait.

Reply:
Also, can you create an actual tracking issue? We shouldn't just link to this PR.